test/integration/PublishFlows.test.ts

```diff
@@ -198,6 +199,112 @@ describe('Publish tests', async () => {
 
   delay(19000)
 
+  it('should publish a dataset with fixed price (create NFT + Datoken + fixed price) using createAsset() fn', async () => {
+    const fixedPriceDdo: DDO = { ...genericAsset }
+    const ownerAddress = publisherAccount
+    const asset = await createAsset(
+      'test asset',
+      'TEST',
+      ownerAddress,
+      assetUrl,
+      '0x13a9813e7CE3fdc867C23Ef9c084570B7F2abdeC', // template 1 on dev network
```

Comment:
I would a templateID here.
If you want to test with addresses, then read addresses.json and use one of the values.
But never hardcode it